css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
+	margin-bottom: 40px;
+	text-align: center;
+	font-weight: bold;
+	font-family: 'Alice', serif;
+}
+
+section {
```

Comment:
Do you always want to apply this to all sections?  Consider using a class instead.

Reply:
I do thank you, all text must be bold and the sections must be relative in order to add an absololute element as a child.

Reply:
Thank you, can you confidently say that will be true for all section ever?  For example, what if you were to add a footer section, would the same rules apply?